elastic/integrations
# [MySQL][TSDB] Add metric type mapping for performance datastream

Type of change
- Enhancement


## What does this PR do?

Add `metric_type` mapping for Performance datastream of MySQL

## Checklist



## How to test this PR locally

- Install / upgrade the integration package
- Verify the data in dashboard and in discover view. 

## Related issues
- Relates #5277
